config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id", to: "guest_rooms#create"
```

Comment:
A more RESTful option here would be "/guests/:id/rooms" to indicate that you're posting to a path that includes both guests and rooms (it's a joins record, not just a guest record). Otherwise, this path makes it seem like it's just creating another guest. I like that you created a joins controller though 👍